modules/olm-updating-index-image.adoc

```diff
@@ -39,9 +49,27 @@ $ opm index add \
 $ podman push <registry>/<namespace>/<existing_index_image>:<tag>
 ----
 
+ifeval::["{context}" == "olm-restricted-networks"]
+. Follow the steps in the _Mirroring an Operator catalog_ procedure again to mirror the updated content. However, when you get to the step about creating the `ImageContentSourcePolicy` (ICSP) object, use the `oc replace` command instead of the `oc create` command. For example:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc replace -f ./{index-image}-manifests/imageContentSourcePolicy.yaml
```

Comment:
It should be -f `oc replace -f ./manifests-{index-image}-<random_number>/imageContentSourcePolicy.yaml`.